bee-network/src/config.rs

```diff
@@ -261,6 +259,8 @@ impl NetworkConfigBuilder {
     }
 
     /// Specifies an interval in seconds for automatic reconnects.
+    ///
+    /// The minimum `secs`
```

Comment:
I don't understand this comment

Reply:
me neither (but the reason probably is that I got distracted while starting that sentence) :trollface: